templates/site/js/node.js

```diff
@@ -144,34 +132,39 @@ document.addEventListener("DOMContentLoaded", function () {
     });
 })
 
-//node name to lowercase
-//build string for class name
-//assign class as needed?
-//store class name in array?
 function getNode(node_id){
     const githubURL = "{{ node_registry_url }}";
-
+    const imageDivClasses = ["d-flex", "flex-column", "node-image-background"];
     fetch(githubURL).then(resp => resp.json()).then(json => {
 
         const node_info = json[node_id];
         const node_keys = Object.keys(json);
 
-        let image = document.getElementById("nodeImage");
-        image.src = nodeImageBackground[node_keys.indexOf(node_id) % nodeImageBackground.length];
+        let imageDiv = document.getElementById("nodeImageDiv");
+        imageDivClasses.push(nodeBackgroundClass[node_keys.indexOf(node_id) % nodeBackgroundClass.length]);
+        let currentImageDivClasses = imageDiv.classList;
+        imageDiv.classList.remove(...currentImageDivClasses);
+        imageDiv.classList.add(...imageDivClasses);
 
         Object.entries(node_info).forEach(([key, val]) => {
             const elem = document.getElementById(key);
             const converted_val = (converters[key] || converters["_default"])(val, node_info)
 
             if (elem !== null) {
-
                 elem.replaceChildren(converted_val);
-
-
             }
         })
 
         let services_title = document.getElementById("servicesTitle");
         services_title.innerText = `Explore All Services by ${node_info["name"]}`;
     })
 }
+
+function capitalize(word){
+    const first_letter = word.charAt(0);
```

Comment:
This whole function can be simplified to:

```javascript
return str.charAt(0).toUpperCase() + str.slice(1)
```

Reply:
Changed to simplified function.